src/lib/utilities.js

```diff
@@ -278,3 +278,6 @@ export function filterByEnvironment(development, beta, production) {
   }
   return development;
 }
+
+export const robotsTxt = 'User-agent: *\n Disallow: /';
```

Comment:
I'll just look into whether we want to do it exactly like this now, I'm a bit in doubt myself exactly how we want to structure this

Reply:
Okay, I figured it out, will also comment further down. Firstly, I noticed, just for semantics (and I checked what they do at `google.com/robots.txt`), delete the space after `\n` and add one at the end of the string.